### PR TITLE
Added documentation about 413 errors with an nginx solution.

### PR DESCRIPTION
**Pull request by avahe-kellenberger**
I had in issue uploading large attachments to issues,
and found the problem was the reverse proxy settings which limited upload size.

I've added docs explaining the issue and provided a solution (for nginx, at least).

---

Feel free to change anything,
or request any changes.

### CODE REVIEW

File: docs/content/doc/usage/reverse-proxies.en-us.md [hunk 1]
```diff
@@ -120,6 +120,17 @@ server {
...
 
+## Resolving Error: 413 Request Entity Too Large
+
+This error indicates nginx is configured to restrict the file upload size.
+
+In your `nginx.conf`,
+find the `http { ... }` block and add the line:
+`client_max_body_size 16M` to set this limit to 16 megabytes.
+
+The 16 can be swapped out the for number of your chosing.
```
Comment (silverwind) [inline docs/content/doc/usage/reverse-proxies.en-us.md]:
```suggestion
In your `nginx.conf`, find the `http { ... }` block and add the line
`client_max_body_size 16M;` to set this limit to 16 megabytes or any other number of choice.
```

**Review by silverwind — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

File: docs/content/doc/usage/reverse-proxies.en-us.md [hunk 2]
```diff
@@ -120,6 +120,14 @@ server {
 }
 ```
 
+## Resolving Error: 413 Request Entity Too Large
+
+This error indicates nginx is configured to restrict the file upload size.
+
+In your `nginx.conf`, find the `http { ... }` block and add the line
```
Comment (lafriks) [inline docs/content/doc/usage/reverse-proxies.en-us.md]: This limit can be set per site and I would actually not recommend changing global setting

**Review by lafriks — DISMISSED** [review]
As per my comment

File: docs/content/doc/usage/reverse-proxies.en-us.md (hunk 2 above)
Comment (silverwind) [reply docs/content/doc/usage/reverse-proxies.en-us.md]: It's allowed in contexts http, server, location. I guess for simple cases with one site it's preferred to set it on `http` level and we can't really cover all config situations.

Comment (lafriks) [reply docs/content/doc/usage/reverse-proxies.en-us.md]: Yes but server should be used in most cases imho in sites-available.d/default conf file

Comment (silverwind) [reply docs/content/doc/usage/reverse-proxies.en-us.md]: I never use those site files on single-site setups, but yeah I guess you have a point as that site-specifc config is the recommended variant.

**Comment by 6543** [comment]
@avahe-kellenberger can you update as suggested :)

File: docs/content/doc/usage/reverse-proxies.en-us.md (hunk 2 above)
Comment (noerw) [reply docs/content/doc/usage/reverse-proxies.en-us.md]:
```suggestion
In your nginx config file containing your Gitea proxy directive, find the `location { ... }` block for Gitea and add the line
```

**Review by 6543 — APPROVED** [review]

**Pull request merged**